app/validators/claim/base_claim_validator.rb

```diff
@@ -372,4 +372,12 @@ def has_higher_rate_mileage?
   def increased_travel?
     @record.expenses.find { |x| x.calculated_distance && x.distance && (x.distance > x.calculated_distance) }
   end
+
+  def validate_earliest_representation_order_date
+    return unless @record.case_type&.name == 'Elected cases not proceeded'
+    return unless @record.earliest_representation_order_date
+    return if @record.earliest_representation_order_date < Settings.agfs_scheme_13_clair_release_date
```

Comment:
I wonder if it's worth noting in a comment that although this uses the `agfs_scheme_13_clair_release_date` for validation, it actually affects both AGFS and LGFS claims? It's possible that someone in future might assume from the code that we're only validating AGFS claims.

Reply:
Good point. LGFS hadn't occurred to me at the time. Strictly it should probably have `if agfs ... elsif lgfs ...` but just having a comment is probably more sensible.